### ISSUE TITLE
Problems with Dedup and meteor-webpack

### ISSUE THREAD

**Issue by fermuch** [thereactivestack/meteor-webpack#42]
Since the last update, `0.4.3`, I'm having this error:

```
Uncaught TypeError: Cannot read property 'apply' of undefined
```

After googling for a bit, I found the problem is because of the Dedup lib of webpack. See https://github.com/webpack/webpack/issues/583

After downgrading to 0.4.2, the problem solved.

**Comment by eXon** [comment]
This might be the case but the error is pretty generic. The 0.4.3 update has nothing to do with the Dedupe plugin. Do you have a full call stack?

**Comment by fermuch** [comment]
This is being originated from the Dedup plugin, since I cloned this package, disabled Dedup and everything is working again. Also, if I work in production mode, this error disappears, because Dedup is not being called on production.

It seems to be some other thing throwing this problem, but I cannot find it. I downgraded several versions backwards, but the problem is still there. It works the first or second time, but after some rebuilds, it begins to fail and I have to delete `.meteor/local`.

The full call stack is this:

![stack trace](http://i.imgur.com/7i1DSeZ.png)


The line throwing the exception:

```javascript
case "object":
	// Module can be created from a template
	modules[moduleId] = (function(_m) {
		var args = _m.slice(1), templateId = _m[0];
		return function (a,b,c) {
			modules[templateId].apply(this, [a,b,c].concat(args)); // <- this line is throwing the exception
		};
	}(_m));
	break;
```

(that's bootstrap.js)
